internal/rootcoord/meta_table.go

```diff
@@ -936,6 +936,54 @@ func (mt *MetaTable) DropIndex(collName, fieldName, indexName string) (typeutil.
 	return dropIdxID, true, nil
 }
 
+func (mt *MetaTable) GetInitBuildIDs(collName, indexName string) ([]UniqueID, error) {
+	mt.ddLock.RLock()
+	defer mt.ddLock.RUnlock()
+
+	collID, ok := mt.collName2ID[collName]
+	if !ok {
+		collID, ok = mt.collAlias2ID[collName]
+		if !ok {
+			return nil, fmt.Errorf("collection not exist with name = %s", collName)
+		}
+	}
+	collMeta, ok := mt.collID2Meta[collID]
+	if !ok {
+		return nil, fmt.Errorf("collection not exist with name = %s", collName)
+	}
```

Comment:
line `943~953` can be simplified by `mt.unlockGetCollectionInfo`.

Reply:
nice, I didn't this method.